packages/graphql/lib/services/resolvers-explorer.service.ts

```diff
@@ -52,6 +54,8 @@ export class ResolversExplorerService extends BaseExplorerService {
     @Inject(GRAPHQL_MODULE_OPTIONS)
     private readonly gqlOptions: GqlModuleOptions,
     private readonly moduleRef: ModuleRef,
+    @Inject(GraphqlContextIdFactory)
```

Comment:
In a final version, this should be removed in favor of the `ContextIdFactory` token, so we can provide an alternative Factory by ourselves using something like:
```ts
const myCustomContextIdFactoryProvider = {
  provide : GraphlqlContextIdFactory,
  useClass: MyCustomClass
}
```